spec/spec_helper.cr

```diff
@@ -38,16 +38,20 @@ DPPM::Log.output = DPPM::Log.error = File.open File::NULL, "w"
 Kemal.config.env = "test"
 FileUtils.mkdir_p Fixtures::DIR.to_s
 at_exit { FileUtils.rm_rf Fixtures::DIR.to_s }
-Fixtures.reset_config
+Fixtures.new.reset_config
 # Run the server
 DppmRestApi.run Socket::IPAddress::LOOPBACK,
   DPPM::Prefix.default_dppm_config.port,
   Fixtures::DIR,
   DPPM::Prefix.new(Fixtures::PREFIX_PATH.to_s).tap &.create
+
+# Having a pool of pre-allocated Fixtures cut to a least 2 times the total execution time
+fixtures_pool = Array(Fixtures).new 999, Fixtures.new
```

Comment:
```suggestion
fixtures_pool = StaticArray(Fixtures, 999).new Fixtures.new
```

Reply:
Was advantage does it bring? As is, `shift` will no longer work.
It could bring some memory and performance improvement, but I doubt there will be significant, especially for specs.

Reply:
It reduces memory consumption. However, if there is a disadvantage (like no shift) then nevermind.

Reply:
After a test, I've realized this approach doesn't work.
Back to the good old `Fixtures.new.reset_config`